forge-gui/res/cardsfolder/g/grafdiggers_cage.txt

```diff
@@ -1,8 +1,8 @@
 Name:Grafdigger's Cage
 ManaCost:1
 Types:Artifact
-R:Event$ Moved | ActiveZones$ Battlefield | Origin$ Graveyard,Library | Destination$ Battlefield | ValidLKI$ Creature.Other | Prevent$ True | Layer$ CantHappen | Description$ Creature cards in graveyards and libraries can't enter the battlefield.
+R:Event$ Moved | ActiveZones$ Battlefield | Origin$ Graveyard,Library | Destination$ Battlefield | ValidLKI$ Creature.Other | Prevent$ True | Layer$ CantHappen | Description$ Creature cards in graveyards and libraries can't enter.
```

Comment:
This seems to be one of those they left unchanged for clarity which feels reasonable

Reply:
Fair. I'll deal with the other instances as well.